MILESTONES.md

```diff
@@ -0,0 +1,50 @@
+# Milestones
```

Comment:
Could you please move the milestones back in the main readme?

One reason is that it provides a high-level overview and makes the readme more self-explanatory.

Additionally, we can have separate .md files for sub-projects that can have their own milestones.